Makefile

```diff
@@ -6,40 +6,61 @@ REGISTRY?=quay.io/coreos/flannel
 # Default tag and architecture. Can be overridden
 TAG?=$(shell git describe --tags --dirty)
 ARCH?=amd64
+# Only enable CGO (and build the UDP backend) on AMD64
+ifeq ($(ARCH),amd64)
+	CGO_ENABLED=1
+else
+	CGO_ENABLED=0
+endif
+
+# Go version to use for builds
+GO_VERSION=1.8.3
```

Comment:
any reason we're not using Go 1.9.2?

Reply:
I don't want to introduce too many changes in this PR. I'll rev the Go version in a different PR